cmake/Platform/Linux/Packaging/snapcraft.yaml.in

```diff
@@ -0,0 +1,64 @@
+name: o3de
+version: '${LY_VERSION_STRING}'
+summary: O3DE Engine
+description: |
+  O3DE Engine
+confinement: classic
+base: core22
+
+parts:
+  test-o3de:
+    plugin: dump
```

Comment:
what is the dump plugin?

Reply:
This plugin basically takes the files as is. The build is leveraging the current build process to build the binaries and generate the install layout, then the dump plugin takes those outputs and packages them. The other option is to let snapcraft also control building O3DE but that's a larger change and this way doesn't stop us from making a linux package from the same build (build once and generate multiple types of packages).